pkg/ccl/changefeedccl/kvfeed/physical_kv_feed.go

```diff
@@ -74,8 +75,13 @@ func (p rangefeedFactory) Run(ctx context.Context, sink kvevent.Writer, cfg rang
 	}
 	g := ctxgroup.WithContext(ctx)
 	g.GoCtx(feed.addEventsToBuffer)
+	var rfOpts []kvcoord.RangeFeedOption
+	if cfg.UseMux {
```

Comment:
I don't think that this is ever set to true. Please add `UseMux:   f.useMux,` to the struct initialization in `kvevent.runUntilTableEvent`.

Reply:
I assume what I said above is the intended behavior. As in, the changefeed should pass the setting down to the underlying rangefeed.